src/samples/audio/nxp_detect_test.c

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright 2021 NXP
+//
+// Author: Cristina Feies <cristina.ilie@nxp.com>
+// Author: Viorel Suman <viorel.suman@nxp.com>
+
+#include <sof/samples/audio/nxp_detect_test.h>
+
+static __aligned(64) uint8_t preprocessed_data[CONFIG_PREPROCESSED_SIZE];
+static __aligned(8) uint8_t confidences[4];
+
+//#define PRINT
+
+int postprocess(uint8_t confidences[4])
```

Comment:
Nonstatic function name is too generic, we might get accidental clashes in the future. nxp_detect_postprocess?